services/management/management.go

```diff
@@ -34,14 +34,27 @@ func nodeID(tx *reform.TX, nodeID, nodeName string, addNodeParams *managementpb.
 	}
 	switch {
 	case nodeID != "":
-		return nodeID, nil
+		node, err := models.FindNodeByID(tx.Querier, nodeID)
+		if err != nil {
+			return "", err
+		}
+		if err = validateExistingNodeType(node); err != nil {
+			return "", err
+		}
+		return node.NodeID, err
 	case nodeName != "":
 		node, err := models.FindNodeByName(tx.Querier, nodeName)
 		if err != nil {
 			return "", err
 		}
-		return node.NodeID, nil
+		if err = validateExistingNodeType(node); err != nil {
+			return "", err
+		}
+		return node.NodeID, err
 	case addNodeParams != nil:
+		if addNodeParams.NodeType != inventorypb.NodeType_REMOTE_NODE {
```

Comment:
According to the ticket, remote RDS nodes should also be accepted

Reply:
@BupycHuk ^

Reply:
got it, will fix it right after DS.

Reply:
@percona-csalguero please request changes for this PR. Otherwise, it lost for everybody:
* it is assigned to you, but you neither merged it, nor declined;
* ticket is also still assigned to you.

Reply:
why do we need RDS node type if we have separate API for RDS?

Reply:
Good point. Do we have a test for that? :)

Reply:
tests for API for RDS or adding node with RDS Node Type?

Reply:
![image](https://user-images.githubusercontent.com/11512/70983487-e45c7000-20c9-11ea-9347-9f8ed70690ad.png)

Reply:
will check and add tests if there are no tests for API for RDS.